<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task5</class>
 <widget class="QMainWindow" name="Task5">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="shopComboBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="shopsCountLineEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>shops count</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createShopsPushButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>create shops</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="productNameLineEdit">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>12</y>
      <width>186</width>
      <height>26</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>product name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addProductPushButton">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>10</y>
      <width>186</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>add product</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getStatisticsPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>401</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>get statistics</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="productsInAllShopsPlainTextEdit">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>101</y>
      <width>121</width>
      <height>192</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="productsInShopsSplainTextEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>121</width>
      <height>192</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="productsNotInShopsPlainTextEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>121</width>
      <height>192</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>in all shops</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>in &gt;= 1 shop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>not in any shop</string>
    </property>
   </widget>
   <widget class="QListWidget" name="productsInShopListWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>121</width>
      <height>192</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>80</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>products in shop</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
